### PR TITLE
Track 4

### PR DESCRIPTION
**Pull request by MatthieuDartiailh**
This is a first version for track 4, that aims at covering how to incorporate noise in a qiskit simulation and how to characterize noise in a qubit.

As with track 3 the directions are rather vague, and I welcome all suggestions.

### CODE REVIEW

File: tracks/tr4_noise.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+Rigetti computing similarly to IBM provides a cloud service for quantum
+computation. The documentation of PyQUIL ("equivalent" of Qiskit ) provides an
+introduction to noise modelling.
+http://docs.rigetti.com/en/stable/noise.html
+
+Finally the chapter 8 of
+[Quantum Computation and Quantum Information](http://mmrc.amss.cas.cn/tlb/201702/W020170224608149940643.pdf)
+covers in a more technical manner the different ways one can represent noise.
+It uses same terminology as Qiskit and may help clarify the Qiskit
```
Comment (joeyuan19) [inline tracks/tr4_noise.md]: *uses **the** same

File: tracks/tr4_noise.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+documentation.
+
+
+## Qiskit
+
+Qiskit provides both tools to simulate the impact of noise on a qubit under Aer
+and tools to characterize qubits under Ignis. Because those characterizations
+require to run many circuits it is not always practical to run them on the real
+hardware, but you can try to run a couple of a real backend to get a feel for
```
Comment (joeyuan19) [inline tracks/tr4_noise.md]: *run a couple **on** a real backend

File: tracks/tr4_noise.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+the real devices limitations.
+
+The following two notebooks cover how to define noise in qiskit, and cover in
+particular the noise model that can be used to approximate the noise on a real
+device.
+https://github.com/Qiskit/qiskit-tutorials/blob/master/qiskit/aer/building_noise_models.ipynb
+https://github.com/Qiskit/qiskit-tutorials/blob/master/qiskit/aer/device_noise_simulation.ipynb
+
+The first Ignis notebook focus on the usual measurements to characterize noise,
```
Comment (joeyuan19) [inline tracks/tr4_noise.md]: notebook **focuses**

**Review by joeyuan19 — COMMENTED** [review]
Some small corrections noted in comments inline.  Looks good.

**Comment by MatthieuDartiailh** [comment]
I addressed all comments.

**Pull request merged**